doc/build/changelog/migration_13.rst

```diff
@@ -454,9 +454,9 @@ The :class:`.AssociationProxy` object makes lots of decisions based on the
 parent mapped class it is associated with.   While the
 :class:`.AssociationProxy` historically began as a relatively simple "getter",
 it became apparent early on that it also needed to make decisions about what
-kind of attribute it is referring towards, e.g. scalar or collection, mapped
+kind of attribute it is referring to, e.g. scalar or collection, mapped
```

Comment:
dp

Reply:
e.g. 

"it became apparent early on that it also needed to make decisions about what kind of attribute to which it refers"

something like that